src/main.cpp

```diff
@@ -2383,6 +2393,27 @@ void loopLED() {
     }
 }
 
+void loopWATER() {
+        if (digitalRead(PIN_WATERSENSOR) == HIGH) {
```

Comment:
No bounce time defined

Reply:
debounced - in principle correct, this is the first version just for showing the icon, so i thought debouncing is not needed. But when testing with the serial output debouncing is however really not needed, water level not really changes, its very stable and may change when you push the water canister away from the sensor, but comes back once the water canister is back in place.

Reply:
Checked again - no debounce needed at all

Reply:
Will debounce when implementing the state change (see comment below)

Reply:
Debounce implemented, PNP  implemented